contributors/design-proposals/storage/raw-block-pv.md

```diff
@@ -528,6 +528,40 @@ spec:
         name: my-nginx-data 
 	readOnly: false
 ```
+## UC9:
+
+DESCRIPTION:
+* A user wishes to read data from a read-only raw block device, such as a database for analytics processing. 
+
+ADMIN:
+* Admin creates a 1 block devices and sets container runtime to read-only mode using the --device option.
```

Comment:
Why does the admin do it?  Isn't Kubernetes going to pass the read only flag to the runtime?

Reply:
@msau42 yes essentially the admin will just create the PV with the accessMode set to RO so the intention it set ahead of time, but you are correct we won't get the runtime flag until we receive the readOnly bit from the PVCVolumeSource

Reply:
The PV access mode is unrelated to the the readonly mode passed to the device.